src/main/java/ReadFile.java

```diff
@@ -0,0 +1,53 @@
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.List;
+
+public class ReadFile {
+
+    public static void main(String... args) {
+        List<Food> foodItems = readFoodFromCSV("sample2.txt"); // let's print all the person read from CSV file
+        for (Food b : foodItems) { System.out.println(b);}}
+
+
+
+        // let's print all the person read from CSV file for (Book b : books) { System.out.println(b); } }
+       private static List<Food> readFoodFromCSV(String fileName) {
+           List<Food> foodList = new ArrayList<>();
+           Path pathToFile = Paths.get(fileName);
+           // create an instance of BufferedReader
+           // using try with resource, Java 7 feature to close resources
+           try (BufferedReader br = Files.newBufferedReader(pathToFile, StandardCharsets.US_ASCII)) {
+               String line = br.readLine();
+
+               while (line != null) {
+                   String[] attributes = line.split(",");
+                   Food foodItem = createFood(attributes);
+                   foodList.add(foodItem);
+                   line = br.readLine(); //read next line before looping
+               }}
+           catch (IOException ioe) { ioe.printStackTrace(); }
+           return foodList;
+
+            }
+
+      private static Food createFood(String[] metadata){
```

Comment:
I don't know if this was the issue, but it might be better if the createFood method is above the readFoodFromCSV method. Because when the program calls createFood on line 29, it will return an error because the method is not defined yet. Try to moving it and see if it changes anything. If it doesn't change, we can always come back later after phase 0 and debug.